contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -89,6 +89,8 @@ contract ValidatorShare is IValidatorShare {
 
     function buyVoucher(uint256 _amount) public onlyWhenUnlocked {
         uint256 share = _amount.mul(100).div(exchangeRate());
+        require(share > 0, "Insufficient amount to buy share ");
```

Comment:
@0xAshish this check doesn't seem right, it should be `_amount > 0`

Reply:
This is fine, id amount is zero then share count will be by default zero.